sapp/ui/trace.py

```diff
@@ -103,6 +120,25 @@ def from_record(
     def is_leaf(self) -> bool:
         return self.callee_port in LEAF_NAMES
 
+    def get_clean_caller(self, tool: str) -> str:
+        return self._clean_callable_name(self.caller, tool)
+
+    def get_clean_callee(self, tool: str) -> str:
+        return self._clean_callable_name(self.callee, tool)
+
+    def _clean_callable_name(self, name: str, tool: str, depth: int = 1) -> str:
```

Comment:
nitpick: Internally we call this "human readable name", could we use the same here?

Also: what does the `depth` parameter do? Maybe remove it since it's never used.

Reply:
I was thinking to use the depth to split not he package_and_class name for something like this 


`A.B.C.D.E.class.function_name` so the depth 0 can only include `class.function_name` and as you increase the depth you get more packages in the name. I know in Java things can become ugly so that's why I thought about the depth. Forgot to use it though 😅

Reply:
> nitpick: Internally we call this "human readable name", could we use the same here?

Gotcha!